packages/app/prisma-utils/src/prismaTransaction.ts

```diff
@@ -72,6 +84,11 @@ const executeTransactionTry = async <T, P extends PrismaClient>(
 	}
 }
 
+const calculateRetryDelay = (retries: number, baseRetryDelayMs: number): number => {
+	// exponential backoff -> 2^(retry-1) * baseRetryDelayMs
+	return Math.pow(2, retries - 1) * baseRetryDelayMs
```

Comment:
maybe we should support optional cap for it too? I reckon that transaction that is executed hours later is not particularly a good idea.

Reply:
Added `maxRetryDelayMs`, it is optional but with a default value of the 30s. let me know if you think the naming or default values are not correct 🙏 

Also, check with Anton and he is fine as long as the retry mechanism follows this rules:
1. Don’t retry forever
2. Don’t create heavy load with retries
3. Skip retries after a reasonable number of tries/time

So I think all good 🎉